Makefile

```diff
@@ -266,6 +269,12 @@ install-minio:
 uninstall-minio:
 	@$(MAKE) run CMD='-c "./build/minio.sh uninstall_minio"'
 
+install-minishift:
+	@/bin/bash ./build/minishift.sh install_minishift $(vm-driver)
```

Comment:
I think the target should be `start-minishift` and `stop-ministift`. Just like kind targets.

Reply:
In the case of `minio` we say install and uninstall minio. Do you think start-minishift, stop-minishift would make more sense.

Reply:
We are doing the same for `kind`. So think it would be better if we have consistent terminology. In case of minio, we are deploying it on K8s, so `install-minio` makes sense